aten/src/ATen/native/cpu/group_norm_kernel.cpp

```diff
@@ -348,8 +347,8 @@ void GroupNormKernelImplChannelsLastInternal(
         T rstd_val = rstd_data[n * G + g];
         for (const auto d : c10::irange(D)) {
           const int64_t c = g * D + d;
-          scale_ptr[c] = rstd_val * (gamma_null ? T(1) : gamma_data[c]);
-          bias_ptr[c] = -scale_ptr[c] * mean_val + (beta_null ? T(0) : beta_data[c]);
+          scale_ptr[c] = rstd_val * (gamma_null ? PT(1) : gamma_data[c]);
```

Comment:
scale_ptr is type T, why PT(1) is used here?

Reply:
`scale_ptr` is of type `T` but `gamma_data` and `beta_data` is of type `PT`.

when input is bfloat16, the params type for mixed data type will be:

* input is bfloat16, params (gamma/beta) is bfloat16: `T` and `PT` are both `BFloat16`.
* input is bfloat16, params (gamma/beta) is float: `T` is `BFloat16` and `PT` is `float`. And compiler would require that in the phrase of `gamma_null ? PT(1) : gamma_data[c]`, `gamma_data[c]` and 1 should have the same type.